recipes/lame/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "lame" %}
+{% set version = "3.100" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/sourceforge/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e
+  patches:
+    - libmp3lame.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - lame --genre-list testcase.mp3
+
+about:
+  home: https://lame.sourceforge.io/
+  license: LGPL
```

Comment:
Generally we try to include license version here as well (e.g. LGPL 2.1 or LGPL 3 depending, have not checked myself for LAME specifically).